src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9858e22e8a.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+   <main class="main-container">
+     <div class="to-do-container">
+       <h3>Today's To Do <span><i class="fas fa-sync-alt"></i></span></h3>
+       <p><i>Add to your list...</i></p><br>
```

Comment:
- [ ] **Required:** This element should be set up to allow user supply input in a later milestone. Consider using an `input` element rather than the current `p` element.
- [ ] **Required:** The structure should also contain an element to add items. Preferably an `a` or `button` element